### PR TITLE
improve some user-facing strings

### PR DESCRIPTION
**Pull request by patricklodder**
A very small collection of minor textual fixes I saved up over the past few weeks, for user-facing string that I found needed clarification or fixing, to be updated prior to string freeze

- improves conciseness of tips displayed in Qt
- makes the function of the reset button on the importkeydialog describe reality
- describe `-alertnotify` to match the new reality after 565280f4
- make sure that "koinu" is not expressed as "koinus"
- fix typo in advanced send screen

### CODE REVIEW

**Comment by patricklodder** [comment]
Got some additional feedback from a shibe working on translations - included these

File: src/qt/overviewpage.cpp
Comment (chromatic) [inline src/qt/overviewpage.cpp:156]: I don't feel strongly about this, but I agree that users should not share their wallet files or private keys with anyone they don't want to have access to their transactions.

Are we losing any important nuance by dropping the mention of "your private key"?

(I think it's a mistake to get into the nuance of *why* you might provide emergency access to your wallet here, for what it's worth.The phrasing in a tip here is fine with me as it is.)

Comment (patricklodder) [reply src/qt/overviewpage.cpp:156]: There is another tip saying exactly the same thing on line 162 about the private key. I think that either we should keep one that mentions both, or keep two separate tips that anyway get rotated through.

I thought this to be the most elegant solution but can adapt to the former and remove line 162.

Comment (chromatic) [reply src/qt/overviewpage.cpp:156]: No, you're right. It's better as you had it.

**Review by chromatic — APPROVED** [review]
utACK

**Comment by chromatic** [comment]
I want to run a quick build before I merge this, but I'll get to it before the end of the day unless another maintainer besides Patrick does so first.

**Comment by patricklodder** [comment]
`(((` [Merge pull request] `)))` <-- that's the force field I put in place around that tease of a merge button on my end 😂

**Review by chromatic — APPROVED** [review]
All tests pass on 64-bit Linux, and strings look fine as rendered. ACK.

**Pull request merged**
